packages/core/src/index.ts

```diff
@@ -35,8 +31,6 @@ type Enumerate<N extends number, Acc extends number[] = []> = Acc['length'] exte
 
 export type HTTPMethodRange<F extends number, T extends number> = Exclude<Enumerate<T>, Enumerate<F>>;
 
-export { getJSONSchemaDefaults, parseResponse, prepareAuth, prepareParams, prepareServer };
```

Comment:
Not only are these utilities causing issues with the CJS export of this library but also these utilities are never used outside of core internals so they don't need to be exported.